server/api/users.js

```diff
@@ -1,6 +1,66 @@
 const router = require('express').Router();
-const { models } = require('../db');
-const { User } = models;
+const bcrypt = require('bcrypt');
+const { models } = require('../db/index');
+const { User, Session } = models;
+
+router.get('/session/:sessionId', (req, res, next) => {
+  const { sessionId } = req.params;
+  if (!sessionId) {
+    return res.status(401).send('login required');
+  } else {
+    User.findOne({
+      where: {
+        sessionId
+      }
+    })
+      .then(user => {
+        res.status(200).send(user);
+      })
+      .catch(e => {
+        res.status(400);
+        next(e);
+      });
+  }
+});
+
+// log in
+router.post('/login', (req, res, next) => {
+  User.findOne({
+    where: {
+      email: req.body.email
+    }
+  })
+    .then(user => {
+      if (!user) {
+        return res.state(401).send('User not found');
+      } else {
+        bcrypt.compare(req.body.password, user.password, (err, result) => {
+          if (err) {
+            console.log(err);
+          }
+          if (result) {
+            Session.create().then(session => {
```

Comment:
I'm not exactly sure what's happening here, but if a user logs in correctly, we're going to create another Session in the database table?

Reply:
User won't get a new session id until they log in.